lib/pf9-builder/testcase-01-centos.csv

```diff
@@ -0,0 +1,8 @@
+## openstack
+# <hostname>|<image>:<username>|<flavor>|<network[,network,...]>|<key>|<security-group>|<role[,role]>[|<lvm-pv-list>|<volume[,volume]>]
+kvm01|CentOS-7-x86_64-GenericCloud-1802:centos|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af,4bcc7dd1-e8b2-404c-9147-3a8141cf1780|bond_members='["ens192"]' bond_sub_interfaces='[{"vlanid":"243","ip":"172.16.243.11","mask":"255.255.255.0"}]'|cm-master|default|glance,glance-public
+kvm02|CentOS-7-x86_64-GenericCloud-1802:centos|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af,4bcc7dd1-e8b2-404c-9147-3a8141cf1780|bond_members='["ens192"]' bond_sub_interfaces='[{"vlanid":"243","ip":"172.16.243.12","mask":"255.255.255.0"}]'|cm-master|default|glance
+kvm03|CentOS-7-x86_64-GenericCloud-1802:centos|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af,4bcc7dd1-e8b2-404c-9147-3a8141cf1780|bond_members='["ens192"]' bond_sub_interfaces='[{"vlanid":"243","ip":"172.16.243.13","mask":"255.255.255.0"}]'|cm-master|default|dhcp,snat
+kvm04|CentOS-7-x86_64-GenericCloud-1802:centos|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af,4bcc7dd1-e8b2-404c-9147-3a8141cf1780|bond_members='["ens192"]' bond_sub_interfaces='[{"vlanid":"243","ip":"172.16.243.14","mask":"255.255.255.0"}]'|cm-master|default|dhcp,snat
+kvm05|CentOS-7-x86_64-GenericCloud-1802:centos|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af,4bcc7dd1-e8b2-404c-9147-3a8141cf1780|bond_members='["ens192"]' bond_sub_interfaces='[{"vlanid":"243","ip":"172.16.243.15","mask":"255.255.255.0"}]'|cm-master|default|cinder|pvs=["/dev/sdb","/dev/sdc","/dev/sdd","/dev/sde"]|kvm05-vol01:50,kvm05-vol02:50,kvm05-vol03:50,kvm05-vol04:50
+kvm06|CentOS-7-x86_64-GenericCloud-1802:centos|m1.medium|5ce89f0f-16aa-447a-a4db-d2910499d5af,4bcc7dd1-e8b2-404c-9147-3a8141cf1780|bond_members='["ens192"]' bond_sub_interfaces='[{"vlanid":"243","ip":"172.16.243.16","mask":"255.255.255.0"}]'|cm-master|default|cinder|pvs=["/dev/sdb","/dev/sdc","/dev/sdd","/dev/sde"]|kvm06-vol01:50,kvm06-vol02:50,kvm06-vol03:50,kvm06-vol04:50
```

Comment:
this should be pulled out, extra stuff a customer cannot use but we can share amongst ourselves